.github/workflows/ci.yml

```diff
@@ -76,12 +76,16 @@ jobs:
           cmake: ${{ matrix.cmake }}
           ninja: true
           vcpkg: true
-          conan: true
+          conan: false
```

Comment:
You can instead specify the version here. No need for pip

Reply:
done